.vscode/settings.template.json

```diff
@@ -1,12 +1,6 @@
 // Rename this file 'settings.json' or merge its
 // contents into your existing settings.
 {
-    "eslint.validate": [
```

Comment:
With this gone, the template consists solely of suggestions for `typescript.tsdk` values.